src/syrupy/__init__.py

```diff
@@ -42,6 +42,14 @@ def pytest_sessionstart(session: Any) -> None:
     session._syrupy.start()
 
 
+def pytest_collection_modifyitems(session: Any, config: Any, items: List[Any]) -> None:
```

Comment:
let's link the pytest docs: https://docs.pytest.org/en/latest/reference.html#_pytest.hookspec.pytest_collection_modifyitems

Reply:
done